cmd/list.go

```diff
@@ -0,0 +1,280 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/urfave/cli"
+
+	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/apitypes"
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/envelope"
+	"github.com/manifoldco/torus-cli/errs"
+	"github.com/manifoldco/torus-cli/pathexp"
+)
+
+func init() {
+	list := cli.Command{
+		Name:      "list",
+		ArgsUsage: "",
+		Usage:     "List your org's project structure and its secrets.",
+		Category:  "SECRETS",
+		Flags: []cli.Flag{
+			orgFlag("Use this organization.", false),
+			projectFlag("Use this project.", false),
+			envSliceFlag("Use this environment.", false),
+			serviceSliceFlag("Use this service.", "", false),
+			nameFlag("Find secrets with this name."),
```

Comment:
Can you explain your reason for a name flag versus using positional arguments?

Reply:
Actually, it looks like this was forgotten?

Reply:
Woops, yeah it was forgotten. Leftover from an early implementation that used flags instead of arguments.